### DISCUSSION TITLE
June 15th PowerShell Community Call

### DISCUSSION THREAD

**Discussion by StevenBucher98** [PowerShell/PowerShell#19675] [Community Call]
Agenda: 
- PSResourceGet Release @anamnavi  
- 7.4 June release updates @adityapatwardhan 
- Docs update @sdwheeler 
- Build video (https://www.youtube.com/watch?v=QXxir1KqjNU)
- Crescendo Video
- Survey - https://forms.office.com/r/hXQYXyzfeD

**Comment by SteveL-MSFT** [comment]
@JustinGrote will talk to a demo on PowerShell Anywhere using Cloudflare

**Comment by gwojan** [comment]
Has anyone considered the idea of adding explicit line ending support for cmdlets like `Out-File`? Now that I'm living xplat it's getting to be a royal PITA dealing with CRLF on Linux. ⁠ 😉
